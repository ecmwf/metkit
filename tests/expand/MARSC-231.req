retrieve,
area=58./-10./35./80,class=od,date=20231231,expver=1,grid=0.1/0.1,levtype=surf,param=2t,step=0,stream=oper,time=00:00:00/12:00:00,type=fc,target=target.bbafa703-e7da-4383-8c34-d36a7cbee4f5,repres=gg,
target="reference.vEDx0X.data"