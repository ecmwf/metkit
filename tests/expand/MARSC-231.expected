RETRIEVE,
    CLASS      = OD,
    TYPE       = FC,
    STREAM     = OPER,
    EXPVER     = 0001,
    REPRES     = GG,
    LEVTYPE    = SFC,
    PARAM      = 167,
    DATE       = 20231231,
    TIME       = 0000/1200,
    STEP       = 0,
    DOMAIN     = G,
    TARGET     = reference.vEDx0X.data,
    RESOL      = AUTO,
    AREA       = 58/-10/35/80,
    GRID       = .1/.1 